scripts/check-users-in-space-developer-role.ps1

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env pwsh
```

Comment:
Is the hashbang required for PowerShell scripts?  I've never had to add it

Reply:
This is not required. I copied the entire script from [BAT](https://github.com/DFE-Digital/bat-infrastructure/blob/main/scripts/check-users-in-space-developer-role.ps1). Will remove. thanks